internal/configs/test_file.go

```diff
@@ -537,6 +662,143 @@ func decodeTestRunOptionsBlock(block *hcl.Block) (*TestRunOptions, hcl.Diagnosti
 	return &opts, diags
 }
 
+func decodeOverrideResourceBlock(block *hcl.Block, mode addrs.ResourceMode) (*OverrideResource, hcl.Diagnostics) {
+	parseTarget := func(attr *hcl.Attribute) (hcl.Traversal, *addrs.ConfigResource, hcl.Diagnostics) {
+		traversal, traversalDiags := hcl.AbsTraversalForExpr(attr.Expr)
+		diags := traversalDiags
+		if traversalDiags.HasErrors() {
+			return nil, nil, diags
+		}
+
+		configRes, configResDiags := addrs.ParseConfigResource(traversal)
+		diags = append(diags, configResDiags.ToHCL()...)
+		if configResDiags.HasErrors() {
+			return nil, nil, diags
+		}
+
+		return traversal, &configRes, diags
+	}
+
+	res := &OverrideResource{
+		Mode: mode,
+	}
+
+	content, diags := block.Body.Content(overrideResourceBlockSchema)
+
+	if attr, exists := content.Attributes["target"]; exists {
```

Comment:
Isn't the `target` attribute in the block required? Feels like we should throw an error id it's missing/

Reply:
Awesome, missed that :)